### ISSUE TITLE
[Accessibility][Usable]: Focus moves out of the dialog(to the Microsoft logo) upon pressing CTRL+Home instead of moving to the first element of the dialog

### ISSUE THREAD

**Issue by jdhuntington** [OfficeDev/office-ui-fabric-react#10541]
Breakout from #8061

**Comment by jdhuntington** [comment]
Unable to repro. Looks like c0bbb0d61e0da3bccbb78654a6f49e0c56b7996e saw the addition of a `FocusTrapZone` which would fix this behavior.
